servers/src/main/java/tachyon/master/next/block/BlockMaster.java

```diff
@@ -449,4 +470,45 @@ private void processWorkerAddedBlocks(MasterWorkerInfo workerInfo,
       }
     }
   }
+
+  /**
+   * Master info periodical status check.
+   */
+  public final class MasterInfoHeartbeatExecutor implements HeartbeatExecutor {
```

Comment:
I don't think "MasterInfoHeartbeatExecutor" is very descriptive. Maybe "LostWorkerDetectorHeartbeat" or something like that?